### PR TITLE
docs: refine command response readme

### PR DESCRIPTION
**Pull request by MuyangAmigo**
@microsoft/adaptivecards-tools	v1.1.0-rc.0
@microsoft/teamsfx-api	v0.20.7-rc.0
@microsoft/teamsfx-cli	v1.1.3-rc.0
@microsoft/teamsfx-core	v1.17.0-rc.0
@microsoft/teams-manifest	v0.0.8-rc.0
@microsoft/metrics-ts	v0.0.2-rc.0
@microsoft/teamsfx	v1.2.1-rc.0
@microsoft/teamsfx-react	v1.0.6-rc.0
ms-teams-vscode-extension	v4.1.0-rc.0

CLI BREAKING CHANGE:
  

Extension-toolkit BREAKING CHANGE:
 

CLI feat commits:
 feat: up ff8539a61
feat: up 9716235ec
feat(cli): support new/add capability for workflow bot (#6008) 46663f250
feat: up (#5697) c00520609
feat: up (#5677) f4b2f935a
feat: prompt user to upgrade spfx project or toolkit to match spfx ve… (#5642) 0f8ca327c 

Extension-toolkit feat commits:
 feat(vsc): add support for untrusted workspace (#6279) 540807041
feat(vsc): add tutorial for card action response (#6261) 242e92b52
feat: support both spfx 1.15 and 1.16 beta  (#6150) 0d532389e
feat: onboard stock update sample app (#5856) 494f4b6f3
feat(vsc): add preview command in treeview to run F5 directly in A/B test (#5878) d9048ba59
feat(vsc): open created project in new window by default (#5870) 83d3dc316
feat: notification when switch account for local debug in vs code (#5781) cfe223dbc
feat: prompt user to upgrade spfx project or toolkit to match spfx ve… (#5642) 0f8ca327c
feat: enable switch tenant for local debug (#5637) ae549cb45

### CODE REVIEW

File: templates/bot/js/command-and-response/README.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
-# Responding to chat commands
+# Overview of the Command bot template
```
Comment (dooriya) [inline templates/bot/js/command-and-response/README.md:1]: Capitalize the first letter?

File: templates/bot/js/command-and-response/README.md [hunk 2]
```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
...
+1. [Step 1: Add a command definition in manifest](#Step-1-Add-a-command-definition-in-manifest)
+1. [Step 2: Respond with an Adaptive Card](#Step-2-Respond-with-an-Adaptive-Card)
+1. [Step 3: Handle the command](#Step-3-Handle-the-command)
+1. [Step 4: Register the new command](#Step-4-Register-the-new-command)
 
-The default pattern to trigger a command is through a defined keyword. But often times you would want to collect and process additional information retrieved after the trigger keyword. In addition to keyword match, you could also define your trigger pattern with [regular expressions](https://regex101.com/) and match against `message.text` with more controls.
+### Step 1: Add a command definition in manifest
 
-When using regular expressions, any capture group can be found in `message.matches`. Below is an example that uses regular expression to capture strings after `reboot`, for example if user inputs `reboot myMachine`, `message.matches[1]` will capture `myMachine`:
+You can edit the manifest template file `templates\appPackage\manifest.template.json` to include definitions of a command with its title and description in the `commands` array:
```
Comment (dooriya) [inline templates/bot/js/command-and-response/README.md:61]: a command -> a `doSomething` command?
The `helloWorld` command is generated by default, while `doSomething` is added for customization.

File: templates/bot/js/command-and-response/README.md [hunk 3]
```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
...
+    ): Promise<string | Partial<Activity>> {
+        // verify the command arguments which are received from the client if needed.
+        console.log(`Bot received message: ${message.text}`);
+
+        const cardData = {
+          title: "doSomething command is added",
+          body: "Congratulations! You have responded to doSomething command",
+        };
+
+        return MessageBuilder.attachAdaptiveCard(helloWorldCard, cardData);
```
Comment (dooriya) [inline templates/bot/js/command-and-response/README.md:136]: Let's replace the following code line:
```
return MessageBuilder.attachAdaptiveCard(helloWorldCard, cardData);
```
with
```jsvascript
const cardJson = AdaptiveCards.declare(doSomethingCard).render(cardData);
return MessageFactory.attachment(CardFactory.adaptiveCard(cardJson));
```

File: templates/bot/js/command-and-response/README.md [hunk 4]
```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
...
+
+```json
+{
+    "type": "AdaptiveCard",
+    "body": [
+    {
+        "type": "TextBlock",
+        "size": "Medium",
+        "weight": "Bolder",
+        "text": "Your DoSomething Command is added!"
```
Comment (dooriya) [inline templates/bot/js/command-and-response/README.md:92]: `DoSomething` -> `doSomething`, better to use the same case as it is defined in the manifest.

File: templates/bot/js/command-and-response/README.md [hunk 5]
```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
...
+          title: "doSomething command is added",
+          body: "Congratulations! You have responded to doSomething command",
+        };
+
+        return MessageBuilder.attachAdaptiveCard(helloWorldCard, cardData);
+    }    
+}
 
-Often you need to connect to existing APIs in order to retrieve data to send to Teams. Teams Toolkit makes it easy for you to configure and manage authentication for existing APIs. 
+module.exports = {
```
Comment (dooriya) [inline templates/bot/js/command-and-response/README.md:140]: Remove extra empty line here (after `module.exports = {`)?

File: templates/bot/js/command-and-response/README.md (hunk 1 above)
Comment (MuyangAmigo) [reply templates/bot/js/command-and-response/README.md:1]: Was following the pattern from workflow bot

File: templates/bot/js/command-and-response/README.md (hunk 2 above)
Comment (MuyangAmigo) [reply templates/bot/js/command-and-response/README.md:61]: Done

File: templates/bot/js/command-and-response/README.md (hunk 4 above)
Comment (MuyangAmigo) [reply templates/bot/js/command-and-response/README.md:92]: Done

File: templates/bot/js/command-and-response/README.md (hunk 3 above)
Comment (MuyangAmigo) [reply templates/bot/js/command-and-response/README.md:136]: Done

**Review by dooriya — APPROVED** [review]

File: templates/bot/js/command-and-response/README.md [hunk 6]
```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
 | `src/helloworldCommandHandler.js` | The business logic to handle a command |
 | `src/cardModels.js` | The default Adaptive Card data model |
```
Comment (dooriya) [inline templates/bot/js/command-and-response/README.md:48]: We should remove this since no `cardModels.js` for JavaScript project

**Review by dooriya — DISMISSED** [review]

File: templates/bot/js/command-and-response/README.md (hunk 6 above)
Comment (MuyangAmigo) [reply templates/bot/js/command-and-response/README.md]: Done

**Review by dooriya — APPROVED** [review]

**Pull request merged**